webpack.playground.config.js

```diff
@@ -13,6 +13,7 @@ const WIDGET_RC = '.widgetrc';
 
 // run `OKTA_SIW_HOST=0.0.0.0 yarn start --watch` to override the host
 const HOST = process.env.OKTA_SIW_HOST || 'localhost';
+const staticDirs = [PLAYGROUND, TARGET];
```

Comment:
I believe `assets` dir should be also included.
Otherwise [`img`](https://github.com/okta/okta-signin-widget/tree/master/assets/img) dir would not be hosted